adm_program/system/classes/TablePhotos.php

```diff
@@ -220,7 +220,40 @@ public function isVisible()
             return false;
         }
 
-        return true;
+        if($gCurrentUser->isAdministrator()) {
+            error_log("[admin - allow access]");
+            return true;
+        }
+        //echo "pho_id:[".$this->getValue('pho_id')."]";
+        $sql = 'SELECT * 
+                      FROM '.TBL_DATES.'
+                     WHERE dat_photo_id = '.$this->getValue('pho_id');
+        $eventsTiedToPhotoAlbums = $this->db->query($sql)->fetchAll();
+        if(!sizeof($eventsTiedToPhotoAlbums)) {
+            // no event is tid to a phot album - so return true
+//            error_log("[no events have photo id - allow access]");
+            return true;
+        }
+
+        foreach($eventsTiedToPhotoAlbums as $event){
+            $sql = 'SELECT * from ' . TBL_MEMBERS . ' M '.
+            'JOIN ' . TBL_ROLES . ' R ON M.mem_rol_id = R.rol_id '.
```

Comment:
you must not concat each line just start a new line within a string.

Reply:
started each line with a new string